src/Video/ShadowPass.cpp

```diff
@@ -0,0 +1,47 @@
+#include "ShadowPass.hpp"
+#include <GL/glew.h>
+
+namespace Video {
+    ShadowPass::ShadowPass() {
+        InitDephtMap();
+        BindBuffer();
+    }
+    ShadowPass::~ShadowPass() {
+    }
```

Comment:
Memory leak. Framebuffer and texture are never deleted.

Reply:
Fixed